workshop-1.3.py

```diff
@@ -4,39 +4,31 @@
 #
 # Run: python workshop-1.3.py
 
-import json
-
 from bokeh.io import show
-from bokeh.models import GMapPlot, GMapOptions, ColumnDataSource, Circle, Range1d
-
-SAN_DIEGO_COORDINATE = (32.712, -117.1611)
-DEFAULT_ZOOM = 16
+from bokeh.models import GMapPlot, GMapOptions, Range1d
 
-# Read the config.json file
-config = []
-with open('config.json', 'r') as f:
-	config = json.load(f)
-    
+from config import get_config
+from constants import DEFAULT_ZOOM, SAN_DIEGO_COORDINATE
 
 
 lat = SAN_DIEGO_COORDINATE[0]
-long = SAN_DIEGO_COORDINATE[1]
```

Comment:
Replaced everywhere `long` was used since it's a type in Python. Similar changes for `dict`, etc.